pkg/controller/options.go

```diff
@@ -97,3 +101,10 @@ type MetricOptions struct {
 	// MRStateMetrics to use for recording state metrics.
 	MRStateMetrics *statemetrics.MRStateMetrics
 }
+
+// ChangeLogOptions for recording changes to managed resources into the change
+// logs.
+type ChangeLogOptions struct {
+	ChangeLogClient changelogs.ChangeLogServiceClient
+	ProviderVersion string
```

Comment:
Is this really a changelog-specific option? I wonder if anything else might need access to the provider version in future.

Reply:
Elevated this provider version controller option to a higher level in https://github.com/crossplane/crossplane-runtime/pull/754/commits/afb1c6b1148b69d12412523e3133266c8c810ea2